src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -81,9 +81,9 @@ class WorkspaceNewRoomPage extends React.Component {
     /**
      * @param {Object} values - form input values passed by the Form component
      */
-    submit(values) {
-        const policyMembers = _.map(_.keys(this.props.allPolicyMembers[`${ONYXKEYS.COLLECTION.POLICY_MEMBERS}${values.policyID}`]), (accountID) => Number(accountID));
-        Report.addPolicyReport(values.policyID, values.roomName, values.visibility, policyMembers);
+    submit({policyID, roomName, visibility, writeCapability}) {
```

Comment:
I'm not 100% sure, but as far as I remember Expensify doesn't like object destructuring like this, there was even an ESLint rule that forbids it

Reply:
There are Expensify rules about the [Destructuring](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring). Since, these values are not state nor props, I applied destructuring assignment.